src/webextension/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+
+  "description": "WebCompat Go Faster Web Extension",
+  "manifest_version": 2,
+  "name": "webcompat-extension",
+  "version": "1.0",
+
+  "permissions": [
+    "tabs",
+    // TODO: Fill required URL for override rules the same as ua_overrides.js
+    // For tablog.js to have permission to inject script to web console.
+    // Example: "*://www.google.com.tw/*"
```

Comment:
That would mean the script would be injected in all websites, no? Does this have any performance implications?

What I was initially thinking about was a way of setting the list of trusted URLs after we loaded the list of overwrites. Is there any way we can set this during runtime?